contrib/scoverage/test/src/HelloWorldTests.scala

```diff
@@ -115,7 +115,7 @@ trait HelloWorldTests extends utest.TestSuite {
 
             assert(
               result.path.toIO.getPath.replace("""\""", "/").endsWith(
-                "mill/target/workspace/mill/contrib/scoverage/HelloWorldTests/eval/HelloWorld/core/scoverage/data/core/scoverage/data.dest"
+                "mill/target/worksources/mill/contrib/scoverage/HelloWorldTests#HelloWorld/out/HelloWorld/core/scoverage/data/core/scoverage/data.dest"
```

Comment:
This looks a bit fishy to me. IIRC, we use `worksources` only for sources (e.g. something to check out only once), but `workspace` to modify as needed. If we now modify `worksources`, this might invalidate various assumptions in other tests and also inpact re-running tests.

Reply:
See my comment above since those 2 are tightly coupled.